src/components/common/view/AddWebApp.web.js

```diff
@@ -188,12 +197,10 @@ const AddWebApp = props => {
       log.debug('Install Prompt fired')
 
       // See if the app is already installed, in that case, do nothing
-      if (
-        (window.matchMedia && window.matchMedia('(display-mode: standalone)').matches) ||
-        window.navigator.standalone === true
-      ) {
+      if (isStandalone) {
```

Comment:
1. why attach event listener at all if in standalone?
2. can we merge the two useeffect on didmount to 1?

Reply:
@sirpy applied your suggestions in a new commit.